core/designsystem/src/main/java/com/b1nd/dodam/designsystem/theme/Typography.kt

```diff
@@ -0,0 +1,150 @@
+package com.b1nd.dodam.designsystem.theme
+
+import androidx.compose.material3.Typography
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+import com.b1nd.dodam.designsystem.R
+
+// Set of Material typography styles to start with
+
+val NotoSansKrFontFamily = FontFamily(
+    Font(R.font.notosanskr_regular, FontWeight.Normal),
+    Font(R.font.notosanskr_medium, FontWeight.Medium),
+    Font(R.font.notosanskr_bold, FontWeight.Bold),
+)
+
+val NotoSansKR =
+    Typography(
+        titleLarge = TextStyle(
+            fontFamily = NotoSansKrFontFamily,
+            fontWeight = FontWeight.Bold,
+            fontSize = 22.sp,
+            lineHeight = 28.sp,
+            letterSpacing = 0.35.sp,
+        ),
+        titleMedium = TextStyle(
+            fontFamily = NotoSansKrFontFamily,
+            fontWeight = FontWeight.Bold,
+            fontSize = 18.sp,
+            lineHeight = 23.sp,
+            letterSpacing = 0.38.sp,
+        ),
+        titleSmall = TextStyle(
+            fontFamily = NotoSansKrFontFamily,
+            fontWeight = FontWeight.Bold,
+            fontSize = 16.sp,
+            lineHeight = 21.sp,
+            letterSpacing = 0.15.sp,
+        ),
+        labelLarge = TextStyle(
+            fontFamily = NotoSansKrFontFamily,
+            fontWeight = FontWeight.Medium,
+            fontSize = 14.sp,
+            lineHeight = 20.sp,
+            letterSpacing = 0.1.sp,
+        ),
+        labelMedium = TextStyle(
+            fontFamily = NotoSansKrFontFamily,
+            fontWeight = FontWeight.Medium,
+            fontSize = 12.sp,
+            lineHeight = 16.sp,
+            letterSpacing = 0.5.sp,
+        ),
+        labelSmall = TextStyle(
+            fontFamily = NotoSansKrFontFamily,
+            fontWeight = FontWeight.Normal,
+            fontSize = 12.sp,
+            lineHeight = 16.sp,
+            letterSpacing = 0.5.sp,
+        ),
+        bodyLarge = TextStyle(
+            fontFamily = NotoSansKrFontFamily,
+            fontWeight = FontWeight.Medium,
+            fontSize = 18.sp,
+            lineHeight = 26.sp,
+            letterSpacing = 0.15.sp,
+        ),
+        bodyMedium = TextStyle(
+            fontFamily = NotoSansKrFontFamily,
+            fontWeight = FontWeight.Medium,
+            fontSize = 16.sp,
+            lineHeight = 24.sp,
+            letterSpacing = 0.5.sp,
+        ),
+        bodySmall = TextStyle(
+            fontFamily = NotoSansKrFontFamily,
+            fontWeight = FontWeight.Medium,
+            fontSize = 14.sp,
+            lineHeight = 20.sp,
+            letterSpacing = 0.25.sp,
+        ),
+    )
+
+val Roboto = Typography(
```

Comment:
I don't think I can't use this font because the theme uses only one font.
DodamDodam uses korean so it would be better to use NotoSans KR only